### DISCUSSION TITLE
Can't Create Database Instance when loading External Module

### DISCUSSION THREAD

**Discussion by dtquocbao** [oqtane/oqtane.framework#1348] [Q&A]
Hello,

I am using Oqtane 2.1.0, creating an external module using version 2.0.2 then rebuild module solution and try to run Oqtane solution to activate it but keep getting this issue. Any advice please?

![image](https://user-images.githubusercontent.com/4122092/118170101-9216a000-b3de-11eb-956d-fce8df9880a5.png)

Thanks

**Comment by leigh-pointer** [comment]
Hi @rockydant 
You need to update your appsettings.config to something like the 

```
{
  "Database": {
    "DatabaseType": "Oqtane.Database.SqlServer.SqlServerDatabase, Oqtane.Database.SqlServer"
  },
  "ConnectionStrings": {
    "DefaultConnection": "Data Sourc
```

2.1.0  is now supporting mulitple DBs

**Reply by dtquocbao** [reply]
Thanks for quick response @leigh-pointer,

This is original database settings in my appsettings.json after I init Oqtane app

![image](https://user-images.githubusercontent.com/4122092/118174499-c04aae80-b3e3-11eb-8346-d214e8ad2ec0.png)

I guess it should be fine because I can launch Oqtane app but when I access the module it keeps return null of database type so Oqtane is not able to create new instance.

Updated: I was trying to use your config too but I was not even able to load the Oqtane.

**Reply by dtquocbao** [reply]
Found you guys' discussion here 
https://github.com/oqtane/oqtane.framework/issues/1317
